dashboard/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM node:10.12.0-alpine as client
+
+RUN mkdir -p /home/root/client
+
+WORKDIR /home/root/client/
+
+# Turn down the verbosity to default level.
+ENV NPM_CONFIG_LOGLEVEL warn
+
+COPY client/package.json ./
+
+RUN npm i
+
+COPY client/src ./src
+COPY client/public ./public
+
+RUN npm run build
+
+### Final stage
+
+FROM node:10.12.0-alpine
+
+WORKDIR /root/
+
+RUN addgroup -S app && adduser -S -g app app
+
+RUN apk --no-cache add curl vim \
```

Comment:
is there a reason this `apk` install / `curl` isn't done in the previous build stage and then copied into this final build stage

Reply:
@bartsmykla WDYT?

Reply:
Hi @ewilde huge sorry, for not getting back to you. I'll respond to you today!

Reply:
@ewilde I just understood your point, and I fixed that. Thank you, and sorry for getting back to this so late